include/z64.h

```diff
@@ -701,6 +701,14 @@ typedef struct {
     }                   restrictions;
 } InterfaceContext; // size = 0x270
 
+typedef enum {
+    /* 0 */ PAUSE_BG_PRERENDER_OFF, // Inactive, do nothing.
+    /* 1 */ PAUSE_BG_PRERENDER_DRAW, // The current frame is only drawn for the purpose of serving as the pause background.
+    /* 2 */ PAUSE_BG_PRERENDER_FILTER, // The previous frame was PAUSE_BG_PRERENDER_DRAW, now apply prerender filters.
```

Comment:
From a modding perspective it may help to be slightly more general in naming this state? I suggest `PROCESS` instead of `FILTER` since a filter is just one way in which the background could be processed

Reply:
also agree here. as a practical example, a mod tharo and I work on replaced the filters here with some blurring instead. 

not that every name has to be mod-inclusive, but I think this is a case where being a little bit more generic doesnt hurt anything, processing still tells you what it does (and is arguably a better state-like verb name)